test/startservers.py

```diff
@@ -47,14 +47,17 @@ def start(race_detection):
     signal.signal(signal.SIGTERM, lambda _, __: stop())
     signal.signal(signal.SIGINT, lambda _, __: stop())
     global processes
-    for srv in [
+    forwards = [
             [":19091", "publisher.boulder:9091"],
             [":19092", "va.boulder:9092"],
             [":19093", "ca.boulder:9093"],
             [":19094", "ra.boulder:9094"],
             [":19095", "sa.boulder:9095"],
             [":19096", "ca.boulder:9096"],
-    ]:
+    ]
+    if default_config_dir.startswith("test/config-next"):
```

Comment:
Since the remote VAs don't depend on any new code, we can run them in both config and config-next versions. In the config version, they'll just be ignored.